service/src/main/kotlin/fi/espoo/evaka/placement/PlacementService.kt

```diff
@@ -12,31 +12,42 @@ import fi.espoo.evaka.serviceneed.ServiceNeed
 import fi.espoo.evaka.serviceneed.clearServiceNeedsFromPeriod
 import fi.espoo.evaka.serviceneed.getServiceNeedsByChild
 import fi.espoo.evaka.serviceneed.getServiceNeedsByUnit
+import fi.espoo.evaka.serviceneed.insertServiceNeed
 import fi.espoo.evaka.shared.DaycareId
 import fi.espoo.evaka.shared.GroupId
 import fi.espoo.evaka.shared.GroupPlacementId
 import fi.espoo.evaka.shared.PlacementId
+import fi.espoo.evaka.shared.ServiceNeedOptionId
 import fi.espoo.evaka.shared.auth.AclAuthorization
 import fi.espoo.evaka.shared.db.Database
 import fi.espoo.evaka.shared.db.mapPSQLException
 import fi.espoo.evaka.shared.domain.BadRequest
 import fi.espoo.evaka.shared.domain.Conflict
 import fi.espoo.evaka.shared.domain.DateRange
 import fi.espoo.evaka.shared.domain.FiniteDateRange
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
 import fi.espoo.evaka.shared.domain.NotFound
 import org.jdbi.v3.core.kotlin.mapTo
 import org.jdbi.v3.json.Json
 import java.time.LocalDate
 import java.util.UUID
 
+data class ApplicationServiceNeed(
+    val optionId: ServiceNeedOptionId,
+    val shiftCare: Boolean,
+    val confirmedBy: UUID?,
+    val confirmedAt: HelsinkiDateTime?
```

Comment:
are these ever not null? I assume these could be removed and hardcoded to null on line 65

Reply:
Fixed in https://github.com/espoon-voltti/evaka/pull/1280/commits/3819bf989cf2aa1923c6422de1a0ade7bd26a2cf